src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const changeTable = Array.from(table.rows);
+
+changeTable.forEach((element) => {
+  const secondColumnClone = element.cells[1].cloneNode(true);
```

Comment:
Ensure that each row has at least two cells to avoid 'undefined' errors when accessing 'element.cells[1]'.